datafusion/core/tests/fuzz_cases/window_fuzz.rs

```diff
@@ -208,6 +208,13 @@ fn get_random_function(
                 vec![],
             ),
         );
+        window_fn_map.insert(
```

Comment:
This version was not in the random tests, it can run in streaming fashion also. Hence I added it to random tests also.